clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -329,6 +331,8 @@ public LeaderFollowerStoreIngestionTask(
       for (Map.Entry<String, VeniceViewWriter> viewWriter: viewWriters.entrySet()) {
         if (viewWriter.getValue() instanceof ChangeCaptureViewWriter) {
           tmpValueForHasChangeCaptureViewWriter = true;
+        }
```

Comment:
why move the "break" out of the first if statement? The original if statement with break inside seems to neater. Maybe there is something I missed.

Reply:
Good catch, I originally had some other code here to detect if the SIT also has materialized view writer but that was removed and I forgot to revert this code. Let me fix that.